docs/configuration.md

```diff
@@ -0,0 +1,83 @@
+# Epinio, CLI configuration
+
+Opinionated platform that runs on Kubernetes, that takes you from App to URL in one step.
+
+![CI](https://github.com/epinio/epinio/workflows/CI/badge.svg)
+
+<img src="./docs/epinio.png" width="50%" height="50%">
+
+The Epinio command line client uses a configuration file to store
+information which has to persist across invokations. This document
+discusses various aspects around this.
+
+## Table Of Contents
+
+  - [Location](#location)
+  - [Contents](#contents)
+  - [Commands](#commands)
+
+## Location
+
+Epinio's configuration is located by default at `~/.config/epinio/config.yaml`.
+
+The location can be changed from command to command by specifying a
+different path with the global command line option `--config-file`.
+
+A more permanent change is possible by setting the environment
+variable `EPINIO_CONFIG` to the desired path.
+
+## Contents
+
+Epinio's configuration contains
+
+  - The name of the organization currently targeted.
+  - Epinio API user name
+  - Epinio API password
+  - Epinio API certificate
+  
+The organization can be changed by running `epoinio target` with the
+name of the desired org as its single argument.
+
+User name and password are used by the client to authenticate itself
+when talking to Epinio's API server. The `epinio install` command
+saves the initial information to the configuration.
+
+The certificate information is set only when installing epinio for
+development. This uses an `omg.howdoi.website` domain with a
+self-signed certificate. `epinio install` saves the associated CA
+certificate to the configuration so that future invokations of the
+client are able to verify the actual certificate when talking to
+Epinio's API server.
+
+## Commands
+
+The Epinio command line client currently provides 3 commands
+explicitly targeting the configuration. These are:
+
+  1. `epinio target`
+
+     As noted in the previous section, this command changes
+     organization to target with all other commands.
+
+  2. `epinio config show`
+
+     This command shows the details of the currently stored
+     configuration settings. An exception is made for the certificate
+     field, due to its expected size. The command's output only notes
+     if certificate data is present or not.
+
+  3. `epinio config update-credentials`
+
+     Epinio allows users to switch between multiple installations (on
+     different clusters) by simply re-targeting the cluster to talk to
+     via changes to the  environment variable `KUBE_CONFIG`.
```

Comment:
It is `KUBECONFIG` instead of `KUBE_CONFIG` [0]

[0] https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/#set-the-kubeconfig-environment-variable

Reply:
will fix

Reply:
Done, commit 96778d2abd064dbcde9ae3f9b1ae81482219f99e